detekt-api/src/testFixtures/kotlin/io/gitlab/arturbosch/detekt/test/TestFactory.kt

```
@@ -58,15 +58,11 @@ fun createRuleInstance(
     id: String = "TestSmell/id",
```

is this default value ever used? it looks strange as if a `$` is missing. Should we choose a different "singleton" value?

---

Yes, it is used in multiple places. A rule id can have two "structures": "RuleSmell" or "RuleSmell/something". "something" alone is not a valid id.

Probably this default value is misleading. I could change it for "TestSmell/postfix" or something similar to avoid confusions. What do you think?